app/models.py

```diff
@@ -4,7 +4,8 @@
 SQLAlchemy models for the app
 '''
 
-from app import ORG_TYPES, VALID_SKILL_LEVELS, QUESTIONS_BY_ID, LEVELS
+from app import ORG_TYPES, VALID_SKILL_LEVELS, QUESTIONS_BY_ID, LEVELS, \
+                QUESTIONNAIRES
```

Comment:
Picky style thing, but I've been using parentheses for long import lines:

    from foo import (bar, baz,
                                boo, bor)

Perhaps there's a PEP about this that I'm in the wrong on.  If so, I'm down to start switching these over.  If not, let's stick to the way I've been doing it.